pkg/rancher-desktop/main/snapshots/snapshots.ts

```diff
@@ -47,7 +52,18 @@ class SnapshotsImpl {
   }
 
   async create(snapshot: Snapshot) : Promise<void> {
-    const response = await this.rdctl(['snapshot', 'create', snapshot.name, '--json']);
+    const command = [
+      'snapshot',
+      'create',
+      snapshot.name,
+      '--json',
+    ];
+
+    if (snapshot.description) {
+      command.push(`--description "${ snapshot.description }"`);
```

Comment:
This needs to be 
```
      command.push('--description',  snapshot.description);
```

Because `spawnFile` takes an array and not a single command-line string, you don't have to do any quoting here.

Reply:
Trivial error, thanks Eric.